sdk/identity/azure-identity/src/main/java/com/azure/identity/EnvironmentCredential.java

```diff
@@ -70,8 +70,9 @@
                 } else if (verifyNotNull(certPath)) {
                     // 1.2 Attempt ClientCertificateCredential
                     logger.info("Azure Identity => EnvironmentCredential invoking ClientCertificateCredential");
-                    targetCredential = new ClientCertificateCredential(tenantId, clientId, certPath,
-                        null, identityClientOptions);
+                    ValidationUtil.validateFilePath(getClass().getSimpleName(), certPath, "Pem Certificate Path");
```

Comment:
Do we know for sure its pem cert?
Should we change the property name to -> "Certificate Path" only ?

Reply:
Yes only PEM certs are supported for Environment credential because there's no environment variable support for the pfx password.